lib/stagehand/database.rb

```diff
@@ -82,16 +82,20 @@ def transaction
       return output
     end
 
-    private
-
-    def connect_to(connection_name)
-      ActiveRecord::Base.connection_specification_name = connection_name
+    def set_connection(klass, connection_name)
```

Comment:
@rywall I'm not thrilled about exposing this method, but because we want to convert the staging connection name into "primary" before setting the connection specification, we're forced to make this helper method available. We could have built this functionality into the `connection_specification_name=` method in `active_record_extensions.rb` but I was trying to treat the multi-threaded support portion of that file as groundwork for a rails PR. However, as I'm writing this it occurs to me that I will only be making a Rails PR for `remove_connection`, so the rest of these methods are fair game for "knowing" about Stagehand. So maybe I will move the `connection_pool_name` logic to the `connection_specification_name=` method after all. Let me know what you think.